__tests__/backend/move/moveFolderContents.test.ts

```diff
@@ -28,7 +28,7 @@ test("moveFolderContents works correctly with an empty folder", () => {
     new ErrorLogger_()
   );
 
-  moveFolderContents_(context, false, false);
+  moveFolderContents_(context, false, false, false);
```

Comment:
In this test files, all the test cases with files should have an additional `expect()`

Reply:
@marekdedic sorry, but I don't fully understand you suggestion - could you share a piece of code for this specific line of code or test, please? I don't see any `expect()` for e.g. `copyComments` here so it's hard to understand what do you mean exactly. Also I'm rather into Chicago style TDD so it's a bit difficult for me to digest these mock-based tests..

Reply:
If you look at the source of `moveFolderContents`, it passes the new argument to `moveFile`. In this file, on lines 69 and 72, there were assertions for the `copyComments` variable. Please add one for `moveOnly` as well.

Reply:
Also, if I remember correctly, there should even be a test case that tests that `copyComments` gets passed correctly - could you please add a similar test case for `moveOnly`?